_generated/AzureCLIV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -31,6 +31,8 @@
   "loc.input.help.powerShellIgnoreLASTEXITCODE": "If this is false, the line `if ((Test-Path -LiteralPath variable:\\LASTEXITCODE)) { exit $LASTEXITCODE }` is appended to the end of your script. This will cause the last exit code from an external command to be propagated as the exit code of powershell. Otherwise the line is not appended to the end of your script.",
   "loc.input.label.visibleAzLogin": "az login output visibility",
   "loc.input.help.visibleAzLogin": "If this is set to true, az login command will output to the task. Setting it to false will suppress the az login output",
+  "loc.input.label.keepAzSessionActive": "[Experimental] Keep Azure CLI session active",
+  "loc.input.help.keepAzSessionActive": "When set to true, the task will keep signing into Azure to prevent AADSTS700024 errors when access tokens are requested past IdToken expiry date. It is an EXPERIMENTAL feature that might not work in all cases. Use at your own risk.",
```

Comment:
We could mention here that this input is valid only for WIF connections and the task will fail in case of setting it to true and using non WIF connection.